chartfx-chart/src/main/resources/de/gsi/chart/ui/titled-border.css

```diff
@@ -1,19 +1,18 @@
-
 .bordered-titled-title {
-  -fx-padding: 0 2 0 2;
-  -fx-font-size: 14px;
-  -fx-background-color: -fx-background;
-  -fx-translate-y: -9;
-  -fx-translate-X: +4;
+    -fx-padding: 0 2 0 2;
+    -fx-font-size: 14px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780860)